### ISSUE TITLE
Can't get Hue to work

### ISSUE THREAD

**Issue by pete2583** [JasonPuglisi/descent#163]
Hey, me again. I'm sure it's just me bring stupid again but I can't get to the Hue settings page. 

I've started the app with environment variables with my hue details. 

Going to /app/hue brings up the option to connect my account. I login and accept the permission. It then reroutes me back to /app/hue but the extra settings aren't there. 

Perhaps I've done something wrong? 

Thanks for any help you can provide. Pete.

**Comment by pete2583** [comment]
So it's installed locally and on a server. The call back addresses I used are:

http://Domain:3000/app/hue

and 

http://127.0.0.0.1/app/hue

I'm running it as a service on Ubuntu so I didn't get errors as I haven't set it to log. 

I will run it from cli and see if I get errors but it'll be tomorrow at least until I can get around to that. 

My other thought was does the callback need to be a Https page? I should be able to do that for it though.

**Comment by pete2583** [comment]
So I setup a reverse proxy and had it redirecting to https. I thought it might be the issue because it will carry an auth.

Still didn't work. 

So I loaded your live site up and looked at the callback url you use when setting up the Hue lights. I noticed your link was /app/hue/authorize so I set my callback url to that and surprise surprise it worked. Lol.

Really cool program though so thanks for sharing. I've since forked it and change the layout so it's more tailored to me.
